setup.cfg

```diff
@@ -1,3 +1,40 @@
 [flake8]
 ignore = E128,E811,W503,E203
 max-line-length = 100
+
+[metadata]
+name = linkstatus
+author = Nikhil Dhandre
+author_email = nik.digitronik@live.com
+license = GPLv3
+description = Simple text/ markdown links status checker
+keywords =
+    linkstatus
+    linkchecker
+    link-checker
+    markdown
+    text
+    linklint
+    link
+url = https://github.com/pythonpune/linkstatus
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Natural Language :: English
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Intended Audience :: End Users/Desktop
+    Intended Audience :: Developers
+    Environment :: Console
+
+[options]
+packages = find:
+zip_safe = False
+setup_requires = setuptools_scm
+install_requires = click; markdown; requests
```

Comment:
Optional: I would try to write this in following way
```
install_requires = 
    click
    markdown
    requests
```

Reply:
Done